src/containers/AboutIndex/components/MembersCard.tsx

```diff
@@ -0,0 +1,42 @@
+import { Box, Grid, makeStyles, Typography } from '@material-ui/core';
+import Paper from 'components/layout/Paper';
+import React from 'react';
+import PeopleIcon from '@material-ui/icons/People';
+import PersonIcon from '@material-ui/icons/Person';
+export type MembersCardProps = {
+  members: string[];
+};
+
+const useStyles = makeStyles((theme) => ({
+  icons: {
+    marginRight: theme.spacing(1),
+  },
+}));
+
+const MembersCard = (props: MembersCardProps) => {
```

Comment:
Destrukturer `props` her for å gjøre det mer oversiktig
```suggestion
const MembersCard = ({ members }: MembersCardProps) => {
```